funsor/interpreter.py

```diff
@@ -10,6 +10,7 @@
 from contextlib import contextmanager
 from functools import singledispatch
 
+import jax
```

Comment:
I see this now forces Funsor users to install both PyTorch and JAX. How difficult would it be to make those requirements optional (as many users would have either PyTorch or JAX but not both)?